<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Run on Chrome desktop"
           tests="7"
           failures="1"
           errors="0"
           skipped="0"
           time="17.198">
  <properties />
            <testcase name="TC_02_Sort_Name_Z_To_A" time="0.892" classname="com.nopcommerce.user.Product_User_Nopcommerce">
                <failure type="java.lang.AssertionError"
                  message="expected [true] but found [false]"
              >
<![CDATA[java.lang.AssertionError: expected [true] but found [false]
  org.testng.Assert.fail(Assert.java:96)
  org.testng.Assert.failNotEquals(Assert.java:776)
  org.testng.Assert.assertTrue(Assert.java:44)
  org.testng.Assert.assertTrue(Assert.java:54)
  commons.BaseTest.checkTrue(BaseTest.java:124)
  commons.BaseTest.verifyTrue(BaseTest.java:136)
  com.nopcommerce.user.Product_User_Nopcommerce.TC_02_Sort_Name_Z_To_A(Product_User_Nopcommerce.java:56)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.base/java.lang.reflect.Method.invoke(Method.java:566)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:648)
  org.testng.TestRunner.run(TestRunner.java:505)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
  org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
  org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
  org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
  java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
  java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
  java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
  java.base/java.lang.Thread.run(Thread.java:834)
]]>
      </failure>

          </testcase>
  
  
            <testcase name="TC_03_Sort_Price_Low_To_High" time="0.824" classname="com.nopcommerce.user.Product_User_Nopcommerce" />
                <testcase name="TC_01_Sort_Name_A_To_Z" time="0.854" classname="com.nopcommerce.user.Product_User_Nopcommerce" />
                <testcase name="TC_05_Product_Display_03_Product" time="1.398" classname="com.nopcommerce.user.Product_User_Nopcommerce" />
                <testcase name="TC_07_Product_Display_09_Product" time="5.956" classname="com.nopcommerce.user.Product_User_Nopcommerce" />
                <testcase name="TC_04_Sort_Price_High_To_Low" time="0.832" classname="com.nopcommerce.user.Product_User_Nopcommerce" />
                <testcase name="TC_06_Product_Display_06_Product" time="6.442" classname="com.nopcommerce.user.Product_User_Nopcommerce" />
      
</testsuite>
